### ISSUE TITLE
MonacoDiffEditor crash when props.value change

### ISSUE THREAD

**Issue by czjxy881** [react-monaco-editor/react-monaco-editor#241]
**Describe the bug**
MonacoDiffEditor crash when props.value change .

```
uncaught Error: this.editor.pushUndoStop is not a function

TypeError: this.editor.pushUndoStop is not a function
    at MonacoDiffEditor.componentDidUpdate (umi.js:235615)
    at commitLifeCycles (umi.js:227532)
    at commitLayoutEffects (umi.js:230281)
    at HTMLUnknownElement.callCallback (umi.js:207815)
    at Object.invokeGuardedCallbackDev (umi.js:207865)
    at invokeGuardedCallback (umi.js:207922)
    at commitRootImpl (umi.js:230053)
    at unstable_runWithPriority (umi.js:247742)
    at runWithPriority$2 (umi.js:218773)
    at commitRoot (umi.js:229882)
    at runRootCallback (umi.js:229022)
    at umi.js:218821
```
**To Reproduce**
In example, add an setState Button
```html
<div>
      <button onClick={()=>this.setState({code:"{}"})} type="button">
            Change by setState
      </button>
      <MonacoDiffEditor
          width="800"
          height="300"
          language="json"
          value={code}
          original={original}
          onChange={this.onChange}
      />
</div>
```

**Expected behavior**
https://github.com/react-monaco-editor/react-monaco-editor/blob/a2f29686ee409955459a1d328f7a08ab30fcaf59/src/diff.js#L27 
`this.editor` should be `this.editor.modifiedEditor.pushUndoStop();`

**Environment (please complete the following information):**
 - OS: [e.g. Linux] MacOS
 - Browser [e.g. Firefox, Safari] Chrome
 - version : 0.30.1

- [x] I will try to send a pull request to fix this issue.

**Comment by deadbeef84** [comment]
Thanks for the fix! :+1: However, it seems version 0.30.2 was never published to npm :worried:

**Comment by domoritz** [comment]
`0.30.2` is on NPM now.
